source/guide_hedgedoc.rst

```diff
@@ -0,0 +1,384 @@
+  success Already up-to-date.
+  Done in 1.15s.
+
+
+  Edit the following config file to setup HedgeDoc server and client.
+  Read more info at https://github.com/hedgedoc/hedgedoc#configuration-files
+
+  * config.json           -- HedgeDoc config
+  * .sequelizerc          -- db config
+  [isabell@stardust hedgedoc]$
+
+Database
+--------
+
+Create a Database.
+
+.. code-block:: console
+
+  [isabell@stardust hedgedoc]$ mysql --verbose --execute="CREATE DATABASE ${USER}_hedgedoc"
+  --------------
+  CREATE DATABASE isabell_hedgedoc
+  --------------
+
+  [isabell@stardust hedgedoc]$
+
+Configuration
+=============
+
+First we remove the config file that was generated by the setup because it might be very confusing. It was a copy of ``config.json.example`` if you like to have a look at it.
+
+.. code-block:: console
+
+
+  [isabell@stardust hedgedoc]$ rm --verbose config.json
+  removed ‘config.json’
+  [isabell@stardust hedgedoc]$
+
+.. note::
+
+  You can set here some options in json format you can't yet set with environment variables. For other configuration options, check the `configuration documentation`_.
+
+Create ``~/hedgedoc/config.json`` with the following content:
+
+.. code-block:: json
+
+  {}
+
+Edit the file ``~/hedgedoc/.sequelizerc`` with the following content and adjust the highlighted line.:
+
+.. code-block:: js
+  :emphasize-lines: 7
+
+  var path = require('path');
+
+  module.exports = {
+      'config':          path.resolve('config.json'),
+      'migrations-path': path.resolve('lib', 'migrations'),
+      'models-path':     path.resolve('lib', 'models'),
+      'url':             'mysql://isabell:YourMySQL-Password@localhost:3306/isabell_hedgedoc'
+  }
+
+.. note:: You will also need the highlighted string later at the step for the ``~/etc/services.d/hedgedoc.ini``, so note it.
+
+Generate session secret
+-----------------------
+
+| The cookie session secret is used to sign the session cookie. If none is set, one will randomly be generated on each startup, meaning all your users will be logged out.
+| You will need the highlighted string at the next step for the ``~/etc/services.d/hedgedoc.ini``.
+
+.. code-block:: console
+  :emphasize-lines: 2
+
+  [isabell@stardust ~]$ pwgen 32 1
+  somethingSuperRandom
+  [isabell@stardust ~]$
+
+
+Setup daemon
+------------
+
+.. note::
+
+  You can set a lot of environment variables. For other configuration options, check the `configuration documentation`_. The prefix for the variables may change in a future release because of the renaming of the project.
+
+Create ``~/etc/services.d/hedgedoc.ini`` with the following content:
+
+.. code-block:: ini
+  :emphasize-lines: 7-8
+
+  [program:hedgedoc]
+  environment=
+    NODE_ENV="production",
+    CMD_ALLOW_EMAIL_REGISTER="false",
+    CMD_ALLOW_ANONYMOUS="false",
+    CMD_ALLOW_FREEURL="true",
+    CMD_SESSION_SECRET="somethingSuperRandom",
+    CMD_DB_URL="mysql://isabell:YourMySQL-Password@localhost:3306/isabell_hedgedoc"
```

Comment:
this can be removed, I think.

Reply:
I'll do that when I merge.

Reply:
I don't think so. I had some issues when I played (last year) with the config settings. - The setting here is mainly a replacement for the setting in the `config.json` - The sql setting in the `.sequelizerc` seems to be important for different components (manage_users is one of them). Even the official documentation looks like it has to be set twice... - I just test this again... - nope, this will result in many unhandled errors with ugly stack trace outputs. 

The old guide https://github.com/Uberspace/lab/commit/182b2b171c350aeb05d47d1222dbfb26117a755e had that mysql config even 3 times :laughing: 

I hope the new config system with 2.0 will get rid of all the mess.